frontend/src/components/UsersPosts.js

```diff
@@ -21,9 +21,9 @@ const UserPosts = () => {
   return (
     <>
       <h3>Users posts</h3>
-      {userPosts.map(post => (
-        <Link to={`/posts/${post.id}`} key={post.id}>
-          <p>{post.title}</p>
+      {userPosts.map(({ id, title }) => (
```

Comment:
I like that you destructured objects in map functions. It looks much cleaner now.

Reply:
Nice to hear.